### DISCUSSION TITLE
Plugin execution handlers should also run on Admin API server

### DISCUSSION THREAD

**Discussion by subnetmarco** [Kong/kong#7198] [Ideas and feature requests]
Thus we can re-use the same plugins in the Admin API without reinventing the wheel. As opposed to the server running on port `8000`, the plugin handlers on `8001` will run a pre-defined list of plugins that cannot be changed, including:
- Authentication
- CORS
- IP whitelisting/blacklisting (once #379 has been implemented as a plugin)
- SSL

**Comment by thibaultcha** [comment]
I think I can give this a try for 0.6.0 if we want to.

**Comment by sonicaghi** [comment]
just got another request for this.

**Comment by thomaskvnze** [comment]
+1

**Comment by cludden** [comment]
+1

**Comment by BrianHutchison** [comment]
+1

**Comment by feichaohao** [comment]
+1
